app/bundles/ReportBundle/Views/Report/details_data.html.twig

```diff
@@ -0,0 +1,160 @@
+                                                    {% if cellType == 'datetime' and  cellVal|length == 10 %}
+                                                        {% set cellType = 'date' %}
+                                                    {% endif %}
+                                                    {% if cellVal is not empty %}
+                                                        {% if cellType == 'datetime' %}
+                                                            {% set cellVal = cellVal|date %}
+                                                        {% elseif cellType == 'date' %}
+                                                            {% set cellVal = cellVal|date %}
+                                                        {% else %}
+                                                            {% set cellVal = cellVal|format %}
+                                                        {% endif %}
+                                                    {% endif %}
+                                                    {% if columnKey.link is defined %}
+                                                        <a href="{{ path(columnKey.link, {'objectAction' : 'view', 'objectId' : attribute(row, columnAlias)}) }}"
+                                                           class="label label-success">
+                                                            {{ cellVal }}</a>
+                                                    {% else %}
+                                                        {{ cellVal }}
+                                                    {% endif %}
+                                                </td>
+                                            {% endif %}
+                                        {% endfor %}
+                                    </tr>
+                                {% endfor %}
+                            {% else %}
+                                <tr>
+                                    <td></td>
+                                    {% for key in columnOrder %}
+                                        <td></td>
+                                    {% endfor %}
+                                </tr>
+                            {% endif %}
+                            <tr class="cm-strong">
+                                <td>{{ 'mautic.report.report.groupby.totals'|trans }}</td>
+                                {% set index = 0 %}
+                                {% for key in columnOrder %}
+                                    <td></td>
+                                {% endfor %}
+                                {% if aggregatorCount > 0 %}
+                                    {% for aggregator in aggregatorOrder %}
+                                        <td>
+                                            {% if total.index is defined %}
+                                                {{ total.index|format('text') }}
```

Comment:
The `format` filter is used like the `sprintf` function. Assuming `total.index` has a value like "this is %s", this would evaluate to "this is text". I'm not sure this is what you want in this case.